pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1572,4 +1572,7 @@ self: super: {
   # This is fixed on lens master. Remove this override on assert fail.
   lens = assert super.lens.version == "4.19.2"; doJailbreak (dontCheck super.lens);
 
+  # Test suite fails, upstream not reachable for simple fix (not responsive on github)
```

Comment:
Could you link a PR here where you've tried communicating with them?

Hopefully they'll become responsive sometime in the future.

Reply:
Ah, I've been a bit terse on that. It's not that they didn't respond, instead they say [in the readme](https://github.com/vivid-synth/vivid#vivid---music-and-sound-synthesis-in-haskell) that they do not respond to PRs in general on github.

Development seems to be non-public, supposedly coordinated via a mailing list that has been inactive since months: https://we.lurk.org/hyperkitty/list/haskell-art@we.lurk.org/
I'm currently trying to get in there and find out more details.

Nevertheless, everything compiles and seems to work. Do you want to wait for more details about upstream responsivity?

Reply:
Ah, that's unfortunate.

I think this looks good as-is, then.  I was able to successfully compile on Linux.

Reply:
https://we.lurk.org/hyperkitty/list/haskell-art@we.lurk.org/thread/VSSEFYAQBLRBLZ7WNIWZWQEVG6YH6ZDC/